src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -205,6 +205,26 @@ public override bool Execute ()
 			Log.LogDebugTaskItems ("  LibraryDirectories:", LibraryDirectories);
 			Log.LogDebugTaskItems ("  LibraryJars:", LibraryJars);
 
+			foreach (var issue in DisabledIssuesByVersion) {
+				if (lintToolVersion < issue.Value) {
+					Regex issueReplaceRegex = new Regex ($"{issue.Key}(,)?");
+					if (!string.IsNullOrEmpty (DisabledIssues) && DisabledIssues.Contains (issue.Key)) {
```

Comment:
This block and the following block are largely identical in structure, they just deal with different variables. This should be factored out into a method.